src/toil/utils/__init__.py

```diff
@@ -1 +1,27 @@
 from __future__ import absolute_import
+
+from toil import version
+from toil.lib.bioio import getBasicOptionParser
+import logging
+
+from toil.provisioners.aws import getCurrentAWSZone
+
+logger = logging.getLogger( __name__ )
+
+
+def getBasicProvisionerParser():
+    parser = getBasicOptionParser()
```

Comment:
I would rather pass the parser in as an argument and rename this to `addBasicProvisionerOptions`. That would allow for multiple such functions for shared arguments to cooperate.